src/ray/util/container_util.h

```diff
@@ -93,10 +103,19 @@ std::ostream &operator<<(std::ostream &os, DebugStringWrapper<std::tuple<Ts...>>
   return os;
 }
 
+template <typename T, std::size_t N>
+std::ostream &operator<<(std::ostream &os, DebugStringWrapper<std::array<T, N>> c) {
+  return c.StringifyContainer(os);
+}
 template <typename... Ts>
 std::ostream &operator<<(std::ostream &os, DebugStringWrapper<std::vector<Ts...>> c) {
   return c.StringifyContainer(os);
 }
+template <typename T, std::size_t N>
+std::ostream &operator<<(std::ostream &os,
+                         DebugStringWrapper<absl::InlinedVector<T, N>> c) {
+  return c.StringifyContainer(os);
```

Comment:
ditto, do we have any usage of this specialization?

Reply:
I think we don't have `absl::InlinedVector` anywhere

Reply:
Well, I use `absl::InlinedVector` in daily basis.

Reply:
I personally use it as the default array-like container, whenever I have a reasonable upper bound estimation for element number.

A few useful util could be built upon inline vector as well, for example, use inline vector to mock map.

Reply:
I am confused. I thought this PR is meant for ray-project/ray where there's no usage of absl::InlinedVector, despite you may have used it in your own projects.

Reply:
I'm trying to say, inline vector is such a common container, and we should add debug string for it;
I could remove if you feel strong about it.

Reply:
Agree that we should introduce usage of inline vector to Ray codebase first before adding a debug string, otherwise we are just introducing some dead code to the codebase.

Reply:
let's remove absl::InlinedVector for now and I will merge this PR

Reply:
OK, removed.